akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -452,7 +491,7 @@ sealed abstract class ByteString extends IndexedSeq[Byte] with IndexedSeqOptimiz
    * @param buffer a ByteBuffer to copy bytes to
    * @return the number of bytes actually copied
    */
-  def copyToBuffer(buffer: ByteBuffer): Int = iterator.copyToBuffer(buffer)
+  def copyToBuffer(buffer: ByteBuffer): Int // specialized in sub-classes
```

Comment:
realised this is binary incompatible.
Shall I revert and override in subclasses?

Not that anyone should be extending ByteString though hm